js/components/viewControlsVr.js

```diff
@@ -72,13 +72,13 @@ var ViewControlsVr = React.createClass({
     }
   },
 
-  render: function () {
+  render: function() {
     var isShowing = !!(this.icon && this.icon.name);
 
     return !isShowing ? null :
-      (<div className={classnames("oo-vr-icon-container view-controls", {"oo-vr-icon-container--hidden": !this.props.controlBarVisible})}>
-        <Icon {...this.props} icon={this.backgroundIcon} className={classnames("oo-vr-icon--substrate")}/>
-        <Icon {...this.props} icon={this.icon.name} className={classnames("oo-vr-icon--icon-symbol")}/>
+      (<div className={classnames('oo-vr-icon-container view-controls', {'oo-vr-icon-container--hidden': !this.props.controlBarVisible})}>
```

Comment:
It feels like we had more double quotes than single quotes, should we stick with doubles?

Reply:
![image](https://user-images.githubusercontent.com/19288322/38522453-5aaecf82-3c51-11e8-9d79-22e62b549389.png)
![image](https://user-images.githubusercontent.com/19288322/38522473-649fa106-3c51-11e8-9ec5-012f6160128d.png)

Reply:
![image](https://user-images.githubusercontent.com/19288322/38522570-ab9ba334-3c51-11e8-8026-25b02b029328.png)
![image](https://user-images.githubusercontent.com/19288322/38522578-b403657a-3c51-11e8-986b-9896bdb7e01d.png)

Reply:
That's barely more. I feel like we should use whatever looks more pleasant/convenient. I'm fine with either one.

Reply:
Let's stick with single. I think we have been using those more recently.